src/plugins/safeSnap/components/HandleOutcomeUma.vue

```diff
@@ -1,20 +1,33 @@
-<script setup>
-import Plugin from '../index';
-import networks from '@snapshot-labs/snapshot.js/src/networks.json';
+<script setup lang="ts">
```

Comment:
typescript was not running on this file at all because we needed to add `lang="ts"`